tls/s2n_cipher_preferences.c

```diff
@@ -791,6 +791,26 @@ const struct s2n_cipher_preferences cipher_preferences_kms_pq_tls_1_0_2019_06 =
         .minimum_protocol_version = S2N_TLS10,
 };
 
+struct s2n_cipher_suite *cipher_suites_kms_pq_tls_1_0_2019_11[] = {
```

Comment:
I think this name should be updated to reflect the fact we plan to use it only for testing and something around SIKE only or no BIKE.

Reply:
That's fair. I'll update it.